src/azul/service/manifest_controller.py

```diff
@@ -137,28 +140,47 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            instructions = dedent(f'''
+            {_curl_instructions(os_system='Unix', cmd=f'curl "{location}"')}
+            {_curl_instructions(os_system='Windows', cmd=f'curl.exe "{location}"')}
```

Comment:
I find it worrisome that this omits the `-L` and `-o` flags that are present in the commands returned by `self.service.command_lines`, especially since the comment in `ManifestGenerator.command_lines` indicates that the `-o` flag is necessary to avoid a bug in the Windows implementation of cURL. The ticket also explicitly mentions the `--location` (`-L`) flag so I find it odd that it's not referenced anywhere. But I might be misunderstanding the intent. 

I appreciate that you can't call `self.service.command_lines` from this branch since the `manifest` parameter can't be filled, but I still think some of the duplication should be reduced between the two assignments of the `instructions` variable. A proper solution might involve some non-trivial refactoring of the `ManifestService.command_lines` method, but I'm not sure.

Reply:
At the very least, we certainly don't need a separate call to `dedent` in both branches. I think something like this (rough sketch) would be cleaner:

```
if manifest is None:
    unix_cmd = ...
    windows_cmd = ...
    ...
else:
    unix_cmd = ...
    windows_cmd = ...
    ...
...
instructions = dedent(f'''
_curl_instructions(os='Unix', cmd=unix_cmd)
_curl_instructions(os='Windows', cmd=windows_cmd)
''')
```

Reply:
Also, the wrapping of the triple-quoted string results in trailing _and leading_ newlines in the response body. I'm not sure if this is intentional. 

```
>>> from textwrap import dedent
>>> dedent(f'''
... foo
... bar
... ''')
'\nfoo\nbar\n'
```